cmd/platform/proxy/settings/update.go

```diff
@@ -0,0 +1,94 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdproxysettings
+
+import (
+	"fmt"
+	"path/filepath"
+
+	proxysettingsapi "github.com/elastic/cloud-sdk-go/pkg/api/platformapi/proxyapi/settingsapi"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	sdkcmdutil "github.com/elastic/cloud-sdk-go/pkg/util/cmdutil"
+	"github.com/spf13/cobra"
+
+	cmdutil "github.com/elastic/ecctl/cmd/util"
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+var platformProxySettingsUpdateCmd = &cobra.Command{
+	Use:     "update --file settings.json",
+	Short:   cmdutil.AdminReqDescription("Updates proxies settings"),
+	PreRunE: cobra.ExactArgs(0),
```

Comment:
Given the complexity if this command, it's probably a good idea to have an `Example:` field with some examples :)